homework_04/11_liliya_lyubenova/test_problem.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for problem.py"""
+from problem import Point, Circle, RelativePosition
+
+def test_no_common_points():
+    """First test"""
```

Comment:
You might briefly explain what you are testing in the function description but in the current case the name of the test says it all.